src/front/ticketer/lib/pages/page_organizer_register.dart

```diff
@@ -0,0 +1,253 @@
+          return "Please enter phone number";
+        }
+        return null;
+      },
+    );
+  }
+
+  Widget _submitButton() {
+    return Container(
+        margin: const EdgeInsets.only(top: 15.0),
+        child: ElevatedButton(
+          onPressed: () => submitOrganizerData(),
+          child: const Text("Submit"),
+        ));
+  }
+
+  Widget _getRegisterContent() {
+    return Form(
+      key: _formKey,
+      child: SingleChildScrollView(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            const Text(
+              "Join us as an organizer",
+              style: TextStyle(
+                  fontSize: 22,
+                  fontWeight: FontWeight.w600,
+                  color: Colors.blueAccent),
+            ),
+            _companyNameEntryField(),
+            _companyCityEntryField(),
+            _companyZipCodeEntryField(),
+            _companyStreetEntryField(),
+            _companyDisplayNameEntryField(),
+            _taxTypeRow(),
+            _companyPhoneEntryField(),
+            _submitButton(),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Row _taxTypeRow() {
+    return Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          const Flexible(
+            flex: 3,
+            child: TaxTypeDropdown(),
+          ),
+          const Spacer(),
+          Flexible(
+            flex: 7,
+            child: _companyTaxIdEntryField(),
+          ),
+        ]);
+  }
+
+  Future<void> submitOrganizerData() async {
+    if (_formKey.currentState!.validate()) {
+      showDialog(
+        context: context,
+        builder: (context) {
+          return AlertDialog(
+            title: const Text("Thank you"),
+            content: const Text(
+                "Administrators have received your form. We will try to verify your data as soon as possible."),
+            actions: [
+              ElevatedButton(
+                  onPressed: () => Navigator.pop(context),
+                  child: const Text('OK')),
+            ],
+          );
+        },
+      );
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Register as an organizer"),
+      ),
+      body: Center(
+        child: Container(
+          constraints: const BoxConstraints(minWidth: 200, maxWidth: 400),
+          padding: const EdgeInsets.all(20),
+          child: _getRegisterContent(),
+        ),
+      ),
+    );
+  }
+}
+
+class TaxTypeDropdown extends StatefulWidget {
+  const TaxTypeDropdown({super.key});
+
+  @override
+  State<TaxTypeDropdown> createState() => _TaxTypeDropdownState();
+}
+
+class _TaxTypeDropdownState extends State<TaxTypeDropdown> {
+  static const List<String> list = <String>[
```

Comment:
I think it is a good idea to take this enum into separate file as not to keep **magic numbers** in code. This is somewhat a magic number.